nltk/parse/stanford.py

```diff
@@ -127,23 +127,62 @@ def raw_parse(self, sentence, verbose=False):
         """
         return next(self.raw_parse_sents([sentence], verbose))
 
-    def raw_parse_sents(self, sentences, verbose=False):
-        """
-        Use StanfordParser to parse multiple sentences. Takes multiple sentences as a
-        list of strings.
-        Each sentence will be automatically tokenized and tagged by the Stanford Parser.
+    def raw_parse_sents(
+        self,
+        sentences,
+        verbose=False,
+        tokenize_whitespace=False,
+        sentence_split=False,
+    ):
+        """Use StanfordParser to parse multiple sentences.
 
-        :param sentences: Input sentences to parse
+        Takes multiple sentences as a list of strings. Each sentence will be
+        automatically tokenized and tagged by the Stanford Parser.
+
+        :param sentences: Input sentences to parse.
         :type sentences: list(str)
         :rtype: iter(iter(Tree))
+
         """
-        cmd = [
-            self._MAIN_CLASS,
-            '-model', self.model_path,
-            '-sentences', 'newline',
-            '-outputFormat', self._OUTPUT_FORMAT,
-        ]
-        return self._parse_trees_output(self._execute(cmd, '\n'.join(sentences), verbose))
+        session = requests.Session()
+
+        for sentence in sentences:
+
+            properties = {
+                'annotators': 'tokenize,pos,lemma,ssplit,{parser_annotator}'.format(
+                    parser_annotator=self.parser_annotator,
+                ),
+                'outputFormat': 'json',
+                'ssplit.isOneSentence': 'false' if sentence_split else 'true',
+                # TODO: Does it work?
+                'tokenize.options': 'normalizeParentheses=true,normalizeOtherBrackets=true',
+            }
+
+            if tokenize_whitespace:
+                properties['tokenize.whitespace'] = 'true'
+
+            response = session.post(
+                'http://localhost:9000',
+                params={
+                    'properties': json.dumps(properties),
+                },
+                data=sentence.encode('utf-8'),
+            )
```

Comment:
Port selection is important, this should be a parameter in `__init__()`  too. Port might clash with other server instances.